composer.json

```diff
@@ -71,6 +82,7 @@
         "symfony/asset": "^4.4 || ^5.4",
         "symfony/config": "^4.4 || ^5.4",
         "symfony/console": "^4.4 || ^5.4",
+        "symfony/debug-bundle": "^4.4 || ^5.4",
```

Comment:
This one must be in require-dev, otherwise it will be installed in production envs.

Reply:
Done.